src/mcts/search.h

```diff
@@ -174,6 +174,8 @@ class Search {
   const MoveList searchmoves_;
   const std::chrono::steady_clock::time_point start_time_;
   int64_t initial_visits_;
+  // dtz_position_ must be initialized before root_move_filter_.
+  bool dtz_position_ = false;
```

Comment:
Does it mean that the root is dtz? Then it may make sense to reflect that in the variable name or at least a comment.

Reply:
Renamed to be clearer.